src/layouts/EditContactUs.jsx

```diff
@@ -774,27 +783,49 @@ const EditContactUs = ({ match }) => {
 
   return (
     <>
-      {showDeletedText && !isEmpty(deletedFrontMatter) && (
+      {!isEmpty(deletedFrontMatter) && (
         <GenericWarningModal
+          isOpen={isRemovedOpen}
+          onClose={onRemovedClose}
           displayTitle="Removed content"
           displayText={`Some of your content has been removed as it is incompatible with the new Isomer format. No changes are permanent unless you press Save on the next page.<br/>${displayDeletedFrontMatter(
             deletedFrontMatter
           )}`}
-          onProceed={() => {
-            setShowDeletedText(false)
-          }}
-          proceedText="Acknowledge"
-        />
+        >
+          <Button onClick={onRemovedClose}>Acknowledge</Button>
+        </GenericWarningModal>
       )}
       {itemPendingForDelete.id && (
```

Comment:
same as above, we can inline this into the `isOpen` prop

Reply:
https://github.com/isomerpages/isomercms-frontend/pull/941/commits/513eedac744b2fe3c2814aff9b935982147c46d2